### ISSUE TITLE
More documentation on rules

### ISSUE THREAD

**Issue by ChrisJefferson** [brosell/obsidian-auto-note-organizer#5]
Could there be a bit more documentation on rules? I'm not clear just from the example `tag[nohash]&project[myproj]`, how rules are specified. Is `tag` a hashtag, or an element named `tag` in the frontmatter? I realise fitting all this in the settings box might be hard, but maybe a documentation page?

Also, "create folders if they don't exist" mentions "capture groups", which I assume means I can somehow refer to moving any project to a folder with the name of the project? How's that work (if I'm guessing right)?

**Comment by connradolisboa** [comment]
What is the projects?

**Comment by aspiers** [comment]
I agree, it's hard to understand what this fork does which is different to the original.
